pandas/tests/frame/methods/test_replace.py

```diff
@@ -1363,3 +1363,27 @@ def test_replace_after_convert_dtypes(self):
         result = df.replace(1, 10)
         expected = pd.DataFrame({"grp": [10, 2, 3, 4, 5]}, dtype="Int64")
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "df, to_replace",
+        [
+            (
+                {"one": ["a", "b ", "c"], "two": ["d ", "e ", "f "]},
+                lambda x: x.strip(),
+            ),
+            (
+                {"one": ["a", "b ", "c"], "two": ["d ", "e ", "f "]},
+                type("callable_object", (object,), dict(__call__=lambda x: x.strip())),
```

Comment:
This is practically the same as the lambda right? I think just the former is fine

Reply:
Yes, it is. I can just move the df and lambda to the body.